### PR TITLE
[CM-7] Fix Scrolling For Concept Map

### PR DESCRIPTION
**Pull request by Connor-Bernard**
Screenshot 2024-09-23 at 6 11 56 PM
Screenshot 2024-09-23 at 6 12 05 PM

### CODE REVIEW

File: dbcron/__pycache__/update_db.cpython-38.pyc
Comment (Mehul-Gandhi) [inline dbcron/__pycache__/update_db.cpython-38.pyc:1]: Why does this need to be pushed?

**Review by Mehul-Gandhi — APPROVED** [review]
Looks good to me and it was great learning about useRef in the context of iframes.

File: website/src/views/conceptMap.js [hunk 1]
```diff
@@ -11,7 +11,17 @@ export default function ConceptMap() {
     const [loading, setLoading] = useState(false);
     const [studentMastery, setStudentMastery] = useState('000000');
 
-    const { selectedStudent, setSelectedStudent } = useContext(StudentSelectionContext);
+    const iframeRef = useRef(null);
```
Comment (AanviKothari456) [inline website/src/views/conceptMap.js:14]: what does setting to null do - just curious

Comment (Connor-Bernard) [reply website/src/views/conceptMap.js:14]: That just initializes the reference to null.  Note that the hooks necessarily load before the html/css meaning when we initialize the ref, we don't yet know what it is referring to.  Once we load the component we are rendering (this is also called mounting the component), we now have a value to assign to the ref.  We do this by assigning the special data tag "ref" on the element and assigning it to the value of the ref we want to assign it to.

File: dbcron/__pycache__/update_db.cpython-38.pyc
Comment (Connor-Bernard) [reply dbcron/__pycache__/update_db.cpython-38.pyc:1]: Just some style fixes in the code.  I could (and quite frankly probably should) make a separate PR for it, but small things like this that don't do anything besides style fixes are okay to be lumped in generally.

**Pull request merged**